src/main/java/lk/successStudent/studentManagement/asset/school/service/SchoolService.java

```diff
@@ -1,29 +1,30 @@
 package lk.successStudent.studentManagement.asset.school.service;
 
 
-import lk.successStudent.studentManagement.asset.school.entity.school;
+import lk.successStudent.studentManagement.asset.school.dao.SchoolDao;
+import lk.successStudent.studentManagement.asset.school.entity.School;
 import lk.successStudent.studentManagement.util.interfaces.AbstractService;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 
 @Service
-public abstract class schoolService implements AbstractService<school, Integer> {
```

Comment:
same issue here refer the article 
https://www.javatpoint.com/java-naming-conventions